engines/suppliers/Gemfile.lock

```diff
@@ -2,68 +2,93 @@ PATH
   remote: .
   specs:
     suppliers (0.0.1)
-      rails (~> 4.1.7)
+      rails (~> 4.2.5)
```

Comment:
nope, Gemfile is only used testing the engine against a dummy application. No use in other cases. In all other cases you use `.gemspec` file as in any other external gem.